en/SettingsView/General.md

```diff
@@ -66,42 +67,50 @@ The settings are:
 
 This section defines the set of devices to which *QGroundControl* will auto-connect. 
 
-![Device autoconnect settings](../../assets/settings/settings_view_general_autoconnect_devices.jpg)
+![Device autoconnect settings](../../assets/settings/general/autoconnect_devices.jpg)
 
 Settings include:
-- **Pixhawk**: Autoconnect to Pixhawk-series device
-- **SiK Radio**:  Autoconnect to SiK (Telemetry) radio
-- **PX4 Flow**: Autoconnect to PX4Flow device
-- **Libre Pilot**: Autoconnect to Libre Pilot autopilot
-- **UDP**: Autoconnect to UDP
-- **RTK GPS**: Autoconnect to RTK GPS device
+- **Pixhawk:** Autoconnect to Pixhawk-series device
+- **SiK Radio:** Autoconnect to SiK (Telemetry) radio
+- **PX4 Flow:** Autoconnect to PX4Flow device
+- **Libre Pilot:** Autoconnect to Libre Pilot autopilot
+- **UDP:** Autoconnect to UDP
+- **RTK GPS:** Autoconnect to RTK GPS device
+- **NMEA GPS Device:** Autoconnect to an external GPS device to get ground station position ([see below](#nmea_gps))
 
-## Ground Station GPS (NMEA GPS Device) {#nmea_gps}
 
-The Ground Station GPS section is used to choose the source of GPS data that will be used for the GCS location. When there is a valid position provided by the NMEA GPS device, the location of the ground control station will be displayed on the map with a purple `Q` icon. If the GPS provides a heading, it will be indicated by the `Q` icon. The GCS location is also used to provide follow me support ([follow me](https://docs.px4.io/en/flight_modes/follow_me.html) is currently implemented for PX4 only).
+### Ground Station Location (NMEA GPS Device) {#nmea_gps}
```

Comment:
@Williangalvani This is section to review please.

Reply:
Looks good to me!
@jaxxzer might want to take a look too.

Reply:
lgtm